src/Elasticsearch/Common/EmptyLogger.php

```diff
@@ -32,7 +32,7 @@ class EmptyLogger extends AbstractLogger implements LoggerInterface
     /**
      * {@inheritDoc}
      */
-    public function log($level, $message, array $context = [])
+    public function log($level, $message, array $context = []): void
```

Comment:
note that this is BC break, but given the name of the class, I guess it is not meant to be extended.